Component/Button/button.css

```diff
@@ -0,0 +1,198 @@
+@import url("../../Utility CSS/navbar.css");
+.box {
+  height: 14rem;
+  width: 120rem;
+  justify-content: flex-start;
+  padding: 2rem;
+}
+
+.btn {
+  height: 5rem;
+  width: 10rem;
+  font-size: 1.8rem;
+  margin: 2rem;
+  padding: 4px 6px;
+  border: none;
+  border-radius: 1rem;
+  color:var(--btn-primary-color
+  background-color: none;
+}
+
+.btn:hover,
+.btn-link:hover {
+  filter: brightness(0.9);
+  cursor: pointer;
```

Comment:
it'd be better to put the base button properties like cursor:pointer inside the .btn class, and only have hover effects in hover pseudo-classes

Reply:
done